lib/elixir/lib/module.ex

```diff
@@ -861,6 +861,46 @@ defmodule Module do
     end, tuples)
   end
 
+  @spec make_overridable(module, module) :: :ok
+  def make_overridable(module, behaviour) when is_atom(module) and is_atom(behaviour) do
+    behaviour_definitions = :ets.lookup_element(data_table_for(module), :behaviour, 2)
+
+    error_explanation = cond do
```

Comment:
Please move this to separate function then, so we can do:

    if error_explanation = check_module_for_overridalbe(module, behaviour) do